solutions/security-enforced/variables.tf

```diff
@@ -0,0 +1,274 @@
+########################################################################################################################
+# Common variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API key to use for IBM Cloud."
+  sensitive   = true
+}
+
+variable "existing_resource_group_name" {
+  type        = string
+  description = "The name of an existing resource group in which to provision the Databases for Elasticsearch in."
+  default     = "Default"
+  nullable    = false
+}
+
+variable "region" {
+  type        = string
+  description = "The region in which the Event Notifications resources are provisioned."
+  default     = "us-south"
+  nullable    = false
+}
+
+variable "prefix" {
+  type        = string
+  description = "(Optional) Prefix to add to all resources created by this solution. To not use any prefix value, you can set this value to `null` or an empty string."
```

Comment:
why is this description different to the fully configurable one? Please ensure consistency